### PR TITLE
feat(cloudformation): Service and Checks

### PR DESCRIPTION
**Pull request by jfagoagas**
### Context 

We have to migrate Prowler v2 --> v3

### Description
Service and checks created for the CloudFormation service.

Migrate the following checks:
- check_extra7154 --> cloudformation_stacks_termination_protection_enabled
- check_extra742 --> cloudformation_outputs_find_secrets

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: providers/aws/services/cloudformation/cloudformation_outputs_find_secrets/cloudformation_outputs_find_secrets.metadata.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "Provider": "aws",
+  "CheckID": "cloudformation_outputs_find_secrets",
+  "CheckTitle": "Find secrets in CloudFormation outputs",
+  "CheckType": [],
+  "ServiceName": "cloudformation",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:partition:cloudformation:region:account-id:stack/resource-id",
+  "Severity": "Critical",
```
Comment (MrCloudSec) [inline providers/aws/services/cloudformation/cloudformation_outputs_find_secrets/cloudformation_outputs_find_secrets.metadata.json]:
```suggestion
  "Severity": "critical",
```

File: providers/aws/services/cloudformation/cloudformation_outputs_find_secrets/cloudformation_outputs_find_secrets.metadata.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "ResourceType": "AwsCloudFormationStack",
+  "Description": "Find secrets in CloudFormation outputs",
+  "Risk": "Secrets hardcoded into CloudFormation outputs can be used by malware and bad actors to gain lateral access to other services.",
+  "RelatedUrl": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-secretsmanager-secret-generatesecretstring.html",
+  "Remediation": {
+    "Code": {
+      "CLI": "",
```
Comment (MrCloudSec) [inline providers/aws/services/cloudformation/cloudformation_outputs_find_secrets/cloudformation_outputs_find_secrets.metadata.json]:
```suggestion
      "CLI": "https://docs.bridgecrew.io/docs/bc_aws_secrets_2#cli-command",
```

File: providers/aws/services/cloudformation/cloudformation_stacks_termination_protection_enabled/cloudformation_stacks_termination_protection_enabled_test.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        stack_name = "Test-Stack"
+        cloudformation_client.stacks = [
+            Stack(
+                arn="arn:aws:cloudformation:eu-west-1:123456789012:stack/Test-Stack/796c8d26-b390-41d7-a23c-0702c4e78b60",
+                name=stack_name,
+                outputs="",
+                region=AWS_REGION,
+            )
+        ]
+        cloudformation_client.stacks[0].enable_termination_protection = False
```
Comment (MrCloudSec) [inline providers/aws/services/cloudformation/cloudformation_stacks_termination_protection_enabled/cloudformation_stacks_termination_protection_enabled_test.py:75]: Can you do it with a boto3 call?

**Review by MrCloudSec — CHANGES_REQUESTED** [review]
Good job 🔝  Please, check my comments

File: providers/aws/services/cloudformation/cloudformation_stacks_termination_protection_enabled/cloudformation_stacks_termination_protection_enabled_test.py (hunk 3 above)
Comment (jfagoagas) [reply providers/aws/services/cloudformation/cloudformation_stacks_termination_protection_enabled/cloudformation_stacks_termination_protection_enabled_test.py:75]: Nope, I think `moto` doesn't cover all the parameters in the describe_stacks.

**Review by MrCloudSec — APPROVED** [review]

**Pull request merged**